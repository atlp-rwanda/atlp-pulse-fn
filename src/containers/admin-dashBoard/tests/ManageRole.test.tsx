/* eslint-disable */
import React from 'react';
import { fireEvent, render } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';
import '@testing-library/jest-dom';
import '../../../../test/jest/__mocks__/matchMedia';
import { MemoryRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';

import ManageRole from '../ManagerRoles';

describe('Manage role page tests', () => {
  it('Removes the model on click', () => {
    const deleteRoleModel = jest.fn();
    const { getByTestId } = render(
      <BrowserRouter>
        <ManageRole />
      </BrowserRouter>,
    );

    let deleteManageRole = getByTestId('delete');
    fireEvent.click(deleteManageRole);
    expect(deleteRoleModel).toHaveBeenCalledTimes(0);
  });
  it('Renders manage role', () => {
    const elem = renderer
      .create(
        <MemoryRouter>
          <ManageRole />
        </MemoryRouter>,
      )
      .toJSON();
    expect(elem).toMatchSnapshot();
  });
});
