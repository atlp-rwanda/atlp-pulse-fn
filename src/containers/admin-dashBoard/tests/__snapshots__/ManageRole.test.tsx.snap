// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Manage role page tests Renders manage role 1`] = `
Array [
  <div
    className="h-screen w-screen z-20 bg-black bg-opacity-30 backdrop-blur-sm absolute flex items-center justify-center px-4 hidden"
  >
    <div
      className="bg-white dark:bg-dark-bg w-full sm:w-3/4 md:w-1/2  xl:w-4/12 rounded-lg p-4 pb-8"
    >
      <div
        className="card-title w-full flex  flex-wrap justify-center items-center  "
      >
        <h3
          className="font-bold text-sm dark:text-white text-center w-11/12"
        >
          Add Trainee
        </h3>
        <hr
          className=" bg-primary border-b my-3 w-full"
        />
      </div>
      <div
        className="card-body"
      >
        <form
          className=" py-3 px-8"
        >
          <div
            className="input my-3 h-9 "
          >
            <div
              className="grouped-input flex items-center h-full w-full rounded-md"
            >
              <input
                className="dark:bg-dark-tertiary border border-primary rounded outline-none px-5 font-sans text-xs py-2 w-full"
                name="name"
                placeholder="name"
                type="text"
              />
            </div>
          </div>
          <div
            className="input my-3 h-9 "
          >
            <div
              className="grouped-input flex items-center h-full w-full rounded-md"
            >
              <input
                className="dark:bg-dark-tertiary border border-primary py-2 rounded outline-none px-5 font-sans text-xs w-full"
                name="email"
                placeholder="email"
                type="email"
              />
            </div>
          </div>
          <div
            className="input my-3 h-9 "
          >
            <div
              className="grouped-input flex items-center h-full w-full rounded-md"
            >
              <input
                className="dark:bg-dark-tertiary border border-primary py-2 rounded outline-none px-5 font-sans text-xs w-full"
                name="cohort"
                placeholder="cohort"
                type="text"
              />
            </div>
          </div>
          <div
            className="input my-3 h-9 "
          >
            <div
              className="grouped-input flex items-center h-full w-full rounded-md"
            >
              <input
                className="dark:bg-dark-tertiary border border-primary py-2 rounded outline-none px-5 font-sans text-xs w-full"
                name="program"
                placeholder="program"
                type="text"
              />
            </div>
          </div>
          <div
            className="w-full flex justify-between"
          >
            <button
              className="btn info sm w-[30%] md:w-1/4 text-sm font-sans"
              onClick={[Function]}
              type="button"
            >
               
              Cancel
               
            </button>
            <button
              className="btn primary sm w-[30%] md:w-1/4 text-sm font-sans"
              type="button"
            >
               
              Save
               
            </button>
          </div>
        </form>
      </div>
    </div>
  </div>,
  <div
    className="min-h-screen w-screen z-30 bg-black bg-opacity-30 backdrop-blur-sm fixed flex items-center justify-center px-4 hidden"
  >
    <div
      className="bg-white dark:bg-dark-bg w-full sm:w-3/4 md:w-1/2  xl:w-4/12 rounded-lg p-4 pb-8"
    >
      <div
        className="card-title w-full flex  flex-wrap justify-center items-center  "
      >
        <h3
          className="font-bold text-sm text-gray-700 dark:text-white text-center w-11/12"
        >
          Removerole
        </h3>
        <hr
          className=" bg-primary border-b my-3 w-full"
        />
      </div>
      <div
        className="card-body"
      >
        <form
          className=" py-3 px-8"
        >
          <div>
            <h2
              className="text-base dark:text-white text-center m-4"
            >
              suredelete
            </h2>
          </div>
          <div
            className="w-full flex justify-between"
          >
            <button
              className="btn info sm w-[30%] md:w-1/4 text-sm font-sans"
              data-testid="delete"
              onClick={[Function]}
              type="button"
            >
               
              Cancel
               
            </button>
            <button
              className="btn danger sm w-[30%] md:w-1/4 text-sm font-sans"
              type="button"
            >
               
              Delete
               
            </button>
          </div>
        </form>
      </div>
    </div>
  </div>,
  <div
    className="bg-light-bg dark:bg-dark-frame-bg min-h-screen pb-16"
  >
    <div
      className="px-3 md:px-8"
    >
      <div
        className="bg-white dark:bg-dark-bg shadow-lg lg:px-5 pt-6 md:pb-6 mt-20 lg:mt-28 rounded-md w-full lg:w-[80%] lg:ml-60"
      >
        <div
          className=" flex items-center justify-between pb-6"
        >
          <div>
            <h2
              className="text-gray-800 dark:text-white font-semibold mb-3 px-2"
            >
              Manageaccess
            </h2>
            <div
              className="flex flex-cols items-center w-full justify-around"
            >
              <div
                className="flex cursor-pointer "
              >
                <div
                  className="px-2 py-3 underline text-primary hover:underline hover:text-primary"
                  onClick={[Function]}
                >
                  All
                </div>
                <div
                  className="px-2 py-3 hover:underline hover:text-primary dark:text-white"
                  onClick={[Function]}
                >
                  Admins
                </div>
                <div
                  className="px-2 py-3 hover:underline hover:text-primary dark:text-white"
                  onClick={[Function]}
                >
                  Coordinator
                </div>
                <div
                  className="px-2 py-3 hover:underline hover:text-primary dark:text-white"
                  onClick={[Function]}
                >
                  Trainees
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          className="flex justify-between items-center space-x-8 px-3"
        >
          <div
            className="relative mt-1"
          >
            <div
              className="absolute inset-y-0 left-0 flex items-center pl-3 pointer-events-none"
            >
              <svg
                className="w-5 h-5 text-gray-500 dark:text-white"
                fill="currentColor"
                viewBox="0 0 20 20"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  clipRule="evenodd"
                  d="M8 4a4 4 0 100 8 4 4 0 000-8zM2 8a6 6 0 1110.89 3.476l4.817 4.817a1 1 0 01-1.414 1.414l-4.816-4.816A6 6 0 012 8z"
                  fillRule="evenodd"
                />
              </svg>
            </div>
            <input
              className="bg-gray-50 dark:bg-dark-tertiary border border-gray-300 text-gray-900 text-sm rounded-lg focus:ring-primary focus:border-primary block w-44 md:w-80 pl-10 p-2.5 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-primary dark:focus:border-primary"
              onChange={[Function]}
              placeholder="Search by Name or Status"
              value=""
            />
          </div>
          <button
            className="btn primary lg px-2"
            type="button"
          >
             
            AddMember
             
          </button>
        </div>
        <div>
          <div
            className="-mx-4 sm:-mx-8 px-4 sm:px-8 py-4 overflow-x-auto"
          >
            <div
              className="inline-block w-full lg:min-w-full shadow rounded-lg overflow-hidden"
            >
              <table
                className="min-w-full leading-normal"
              >
                <thead>
                  <tr>
                    <th
                      className="py-6 px-5 md:px-12 border-b-2 border-gray-200 bg-gray-100 dark:bg-dark-tertiary text-left text-xs font-semibold text-gray-600 dark:text-white uppercase tracking-wider"
                    >
                      name
                    </th>
                    <th
                      className="py-[26px] px-5 border-b-2 border-gray-200 bg-gray-100 dark:bg-dark-tertiary text-left text-xs font-semibold text-gray-600 dark:text-white uppercase tracking-wider"
                    >
                      Status
                    </th>
                    <th
                      className="px-5 py-3 border-b-2 border-gray-200 bg-gray-100 dark:bg-dark-tertiary text-left text-xs font-semibold text-gray-600 dark:text-white uppercase tracking-wider"
                    >
                      Role
                    </th>
                    <th
                      className="px-5 py-3 border-b-2 border-gray-200 bg-gray-100 dark:bg-dark-tertiary text-left text-xs font-semibold text-gray-600 dark:text-white uppercase tracking-wider"
                    >
                      Action
                    </th>
                  </tr>
                </thead>
                <tbody>
                  <tr
                    className="bg-white dark:bg-dark-bg"
                  >
                    <td
                      className="border-b border-gray-200 py-2"
                    >
                      <div
                        className="flex items-left"
                      >
                        <span
                          className="hidden  ml-2 md:inline-block h-10 w-10 rounded-full overflow-hidden bg-gray-100 dark:bg-dark-tertiary"
                        >
                          <svg
                            className="h-full w-full text-gray-300 dark:text-white"
                            fill="currentColor"
                            viewBox="0 0 24 24"
                          >
                            <path
                              d="M24 20.993V24H0v-2.996A14.977 14.977 0 0112.004 15c4.904 0 9.26 2.354 11.996 5.993zM16.002 8.999a4 4 0 11-8 0 4 4 0 018 0z"
                            />
                          </svg>
                        </span>
                        <div
                          className="flex flex-col  leading-4 px-3 py-2"
                        >
                          <h3
                            className="dark:text-white sm:text-xs"
                          >
                            Keza Alice
                          </h3>
                          <p
                            className="text-sm sm:text-xs text-gray-400 dark:text-white"
                          >
                            keza@gmail.com
                          </p>
                        </div>
                      </div>
                    </td>
                    <td
                      className="py-3 px-5 border-b border-gray-200  text-sm sm:text-xs"
                    >
                      <p
                        className="text-green-500 whitespace-no-wrap"
                      >
                        Active
                      </p>
                    </td>
                    <td
                      className="px-5 py-3 border-b border-gray-200  text-sm"
                    >
                      <div
                        className="whitespace-no-wrap flex justify-left"
                      >
                        <div
                          className="xl:w-24"
                        >
                          admin
                        </div>
                      </div>
                    </td>
                    <td
                      className="px-5 py-3 border-b border-gray-200  text-[9px] md:text-sm"
                    >
                      <p
                        className="text-red-500 whitespace-no-wrap cursor-pointer"
                        onClick={[Function]}
                      >
                        Remove
                      </p>
                    </td>
                  </tr>
                  <tr
                    className="bg-light-bg dark:bg-dark-tertiary"
                  >
                    <td
                      className="border-b border-gray-200 py-2"
                    >
                      <div
                        className="flex items-left"
                      >
                        <span
                          className="hidden  ml-2 md:inline-block h-10 w-10 rounded-full overflow-hidden bg-gray-100 dark:bg-dark-tertiary"
                        >
                          <svg
                            className="h-full w-full text-gray-300 dark:text-white"
                            fill="currentColor"
                            viewBox="0 0 24 24"
                          >
                            <path
                              d="M24 20.993V24H0v-2.996A14.977 14.977 0 0112.004 15c4.904 0 9.26 2.354 11.996 5.993zM16.002 8.999a4 4 0 11-8 0 4 4 0 018 0z"
                            />
                          </svg>
                        </span>
                        <div
                          className="flex flex-col  leading-4 px-3 py-2"
                        >
                          <h3
                            className="dark:text-white sm:text-xs"
                          >
                            cyifuzo Didider
                          </h3>
                          <p
                            className="text-sm sm:text-xs text-gray-400 dark:text-white"
                          >
                            cyifuzo@gmail.com
                          </p>
                        </div>
                      </div>
                    </td>
                    <td
                      className="py-3 px-5 border-b border-gray-200  text-sm sm:text-xs"
                    >
                      <p
                        className="text-green-500 whitespace-no-wrap"
                      >
                        Active
                      </p>
                    </td>
                    <td
                      className="px-5 py-3 border-b border-gray-200  text-sm"
                    >
                      <div
                        className="whitespace-no-wrap flex justify-left"
                      >
                        <div
                          className="xl:w-24"
                        >
                          admin
                        </div>
                      </div>
                    </td>
                    <td
                      className="px-5 py-3 border-b border-gray-200  text-[9px] md:text-sm"
                    >
                      <p
                        className="text-red-500 whitespace-no-wrap cursor-pointer"
                        onClick={[Function]}
                      >
                        Remove
                      </p>
                    </td>
                  </tr>
                  <tr
                    className="bg-white dark:bg-dark-bg"
                  >
                    <td
                      className="border-b border-gray-200 py-2"
                    >
                      <div
                        className="flex items-left"
                      >
                        <span
                          className="hidden  ml-2 md:inline-block h-10 w-10 rounded-full overflow-hidden bg-gray-100 dark:bg-dark-tertiary"
                        >
                          <svg
                            className="h-full w-full text-gray-300 dark:text-white"
                            fill="currentColor"
                            viewBox="0 0 24 24"
                          >
                            <path
                              d="M24 20.993V24H0v-2.996A14.977 14.977 0 0112.004 15c4.904 0 9.26 2.354 11.996 5.993zM16.002 8.999a4 4 0 11-8 0 4 4 0 018 0z"
                            />
                          </svg>
                        </span>
                        <div
                          className="flex flex-col  leading-4 px-3 py-2"
                        >
                          <h3
                            className="dark:text-white sm:text-xs"
                          >
                            Kaka Makombe
                          </h3>
                          <p
                            className="text-sm sm:text-xs text-gray-400 dark:text-white"
                          >
                            kaka@gmail.com
                          </p>
                        </div>
                      </div>
                    </td>
                    <td
                      className="py-3 px-5 border-b border-gray-200  text-sm sm:text-xs"
                    >
                      <p
                        className="text-green-500 whitespace-no-wrap"
                      >
                        Active
                      </p>
                    </td>
                    <td
                      className="px-5 py-3 border-b border-gray-200  text-sm"
                    >
                      <div
                        className="whitespace-no-wrap flex justify-left"
                      >
                        <div
                          className="xl:w-24"
                        >
                          coordinator
                        </div>
                      </div>
                    </td>
                    <td
                      className="px-5 py-3 border-b border-gray-200  text-[9px] md:text-sm"
                    >
                      <p
                        className="text-red-500 whitespace-no-wrap cursor-pointer"
                        onClick={[Function]}
                      >
                        Remove
                      </p>
                    </td>
                  </tr>
                  <tr
                    className="bg-light-bg dark:bg-dark-tertiary"
                  >
                    <td
                      className="border-b border-gray-200 py-2"
                    >
                      <div
                        className="flex items-left"
                      >
                        <span
                          className="hidden  ml-2 md:inline-block h-10 w-10 rounded-full overflow-hidden bg-gray-100 dark:bg-dark-tertiary"
                        >
                          <svg
                            className="h-full w-full text-gray-300 dark:text-white"
                            fill="currentColor"
                            viewBox="0 0 24 24"
                          >
                            <path
                              d="M24 20.993V24H0v-2.996A14.977 14.977 0 0112.004 15c4.904 0 9.26 2.354 11.996 5.993zM16.002 8.999a4 4 0 11-8 0 4 4 0 018 0z"
                            />
                          </svg>
                        </span>
                        <div
                          className="flex flex-col  leading-4 px-3 py-2"
                        >
                          <h3
                            className="dark:text-white sm:text-xs"
                          >
                            Keza Alice
                          </h3>
                          <p
                            className="text-sm sm:text-xs text-gray-400 dark:text-white"
                          >
                            keza@gmail.com
                          </p>
                        </div>
                      </div>
                    </td>
                    <td
                      className="py-3 px-5 border-b border-gray-200  text-sm sm:text-xs"
                    >
                      <p
                        className="text-green-500 whitespace-no-wrap"
                      >
                        Active
                      </p>
                    </td>
                    <td
                      className="px-5 py-3 border-b border-gray-200  text-sm"
                    >
                      <div
                        className="whitespace-no-wrap flex justify-left"
                      >
                        <div
                          className="xl:w-24"
                        >
                          trainee
                        </div>
                      </div>
                    </td>
                    <td
                      className="px-5 py-3 border-b border-gray-200  text-[9px] md:text-sm"
                    >
                      <p
                        className="text-red-500 whitespace-no-wrap cursor-pointer"
                        onClick={[Function]}
                      >
                        Remove
                      </p>
                    </td>
                  </tr>
                  <tr
                    className="bg-white dark:bg-dark-bg"
                  >
                    <td
                      className="border-b border-gray-200 py-2"
                    >
                      <div
                        className="flex items-left"
                      >
                        <span
                          className="hidden  ml-2 md:inline-block h-10 w-10 rounded-full overflow-hidden bg-gray-100 dark:bg-dark-tertiary"
                        >
                          <svg
                            className="h-full w-full text-gray-300 dark:text-white"
                            fill="currentColor"
                            viewBox="0 0 24 24"
                          >
                            <path
                              d="M24 20.993V24H0v-2.996A14.977 14.977 0 0112.004 15c4.904 0 9.26 2.354 11.996 5.993zM16.002 8.999a4 4 0 11-8 0 4 4 0 018 0z"
                            />
                          </svg>
                        </span>
                        <div
                          className="flex flex-col  leading-4 px-3 py-2"
                        >
                          <h3
                            className="dark:text-white sm:text-xs"
                          >
                            Ella Didider
                          </h3>
                          <p
                            className="text-sm sm:text-xs text-gray-400 dark:text-white"
                          >
                            kdidier@gmail.com
                          </p>
                        </div>
                      </div>
                    </td>
                    <td
                      className="py-3 px-5 border-b border-gray-200  text-sm sm:text-xs"
                    >
                      <p
                        className="text-green-500 whitespace-no-wrap"
                      >
                        Active
                      </p>
                    </td>
                    <td
                      className="px-5 py-3 border-b border-gray-200  text-sm"
                    >
                      <div
                        className="whitespace-no-wrap flex justify-left"
                      >
                        <div
                          className="xl:w-24"
                        >
                          trainee
                        </div>
                      </div>
                    </td>
                    <td
                      className="px-5 py-3 border-b border-gray-200  text-[9px] md:text-sm"
                    >
                      <p
                        className="text-red-500 whitespace-no-wrap cursor-pointer"
                        onClick={[Function]}
                      >
                        Remove
                      </p>
                    </td>
                  </tr>
                  <tr
                    className="bg-light-bg dark:bg-dark-tertiary"
                  >
                    <td
                      className="border-b border-gray-200 py-2"
                    >
                      <div
                        className="flex items-left"
                      >
                        <span
                          className="hidden  ml-2 md:inline-block h-10 w-10 rounded-full overflow-hidden bg-gray-100 dark:bg-dark-tertiary"
                        >
                          <svg
                            className="h-full w-full text-gray-300 dark:text-white"
                            fill="currentColor"
                            viewBox="0 0 24 24"
                          >
                            <path
                              d="M24 20.993V24H0v-2.996A14.977 14.977 0 0112.004 15c4.904 0 9.26 2.354 11.996 5.993zM16.002 8.999a4 4 0 11-8 0 4 4 0 018 0z"
                            />
                          </svg>
                        </span>
                        <div
                          className="flex flex-col  leading-4 px-3 py-2"
                        >
                          <h3
                            className="dark:text-white sm:text-xs"
                          >
                            Kaka Makombe
                          </h3>
                          <p
                            className="text-sm sm:text-xs text-gray-400 dark:text-white"
                          >
                            kaka@gmail.com
                          </p>
                        </div>
                      </div>
                    </td>
                    <td
                      className="py-3 px-5 border-b border-gray-200  text-sm sm:text-xs"
                    >
                      <p
                        className="text-green-500 whitespace-no-wrap"
                      >
                        Active
                      </p>
                    </td>
                    <td
                      className="px-5 py-3 border-b border-gray-200  text-sm"
                    >
                      <div
                        className="whitespace-no-wrap flex justify-left"
                      >
                        <div
                          className="xl:w-24"
                        >
                          coordinator
                        </div>
                      </div>
                    </td>
                    <td
                      className="px-5 py-3 border-b border-gray-200  text-[9px] md:text-sm"
                    >
                      <p
                        className="text-red-500 whitespace-no-wrap cursor-pointer"
                        onClick={[Function]}
                      >
                        Remove
                      </p>
                    </td>
                  </tr>
                  <tr
                    className="bg-white dark:bg-dark-bg"
                  >
                    <td
                      className="border-b border-gray-200 py-2"
                    >
                      <div
                        className="flex items-left"
                      >
                        <span
                          className="hidden  ml-2 md:inline-block h-10 w-10 rounded-full overflow-hidden bg-gray-100 dark:bg-dark-tertiary"
                        >
                          <svg
                            className="h-full w-full text-gray-300 dark:text-white"
                            fill="currentColor"
                            viewBox="0 0 24 24"
                          >
                            <path
                              d="M24 20.993V24H0v-2.996A14.977 14.977 0 0112.004 15c4.904 0 9.26 2.354 11.996 5.993zM16.002 8.999a4 4 0 11-8 0 4 4 0 018 0z"
                            />
                          </svg>
                        </span>
                        <div
                          className="flex flex-col  leading-4 px-3 py-2"
                        >
                          <h3
                            className="dark:text-white sm:text-xs"
                          >
                            Keza Alice
                          </h3>
                          <p
                            className="text-sm sm:text-xs text-gray-400 dark:text-white"
                          >
                            keza@gmail.com
                          </p>
                        </div>
                      </div>
                    </td>
                    <td
                      className="py-3 px-5 border-b border-gray-200  text-sm sm:text-xs"
                    >
                      <p
                        className="text-green-500 whitespace-no-wrap"
                      >
                        Active
                      </p>
                    </td>
                    <td
                      className="px-5 py-3 border-b border-gray-200  text-sm"
                    >
                      <div
                        className="whitespace-no-wrap flex justify-left"
                      >
                        <div
                          className="xl:w-24"
                        >
                          admin
                        </div>
                      </div>
                    </td>
                    <td
                      className="px-5 py-3 border-b border-gray-200  text-[9px] md:text-sm"
                    >
                      <p
                        className="text-red-500 whitespace-no-wrap cursor-pointer"
                        onClick={[Function]}
                      >
                        Remove
                      </p>
                    </td>
                  </tr>
                  <tr
                    className="bg-light-bg dark:bg-dark-tertiary"
                  >
                    <td
                      className="border-b border-gray-200 py-2"
                    >
                      <div
                        className="flex items-left"
                      >
                        <span
                          className="hidden  ml-2 md:inline-block h-10 w-10 rounded-full overflow-hidden bg-gray-100 dark:bg-dark-tertiary"
                        >
                          <svg
                            className="h-full w-full text-gray-300 dark:text-white"
                            fill="currentColor"
                            viewBox="0 0 24 24"
                          >
                            <path
                              d="M24 20.993V24H0v-2.996A14.977 14.977 0 0112.004 15c4.904 0 9.26 2.354 11.996 5.993zM16.002 8.999a4 4 0 11-8 0 4 4 0 018 0z"
                            />
                          </svg>
                        </span>
                        <div
                          className="flex flex-col  leading-4 px-3 py-2"
                        >
                          <h3
                            className="dark:text-white sm:text-xs"
                          >
                            Kamanzi Didider
                          </h3>
                          <p
                            className="text-sm sm:text-xs text-gray-400 dark:text-white"
                          >
                            kdidier@gmail.com
                          </p>
                        </div>
                      </div>
                    </td>
                    <td
                      className="py-3 px-5 border-b border-gray-200  text-sm sm:text-xs"
                    >
                      <p
                        className="text-green-500 whitespace-no-wrap"
                      >
                        Active
                      </p>
                    </td>
                    <td
                      className="px-5 py-3 border-b border-gray-200  text-sm"
                    >
                      <div
                        className="whitespace-no-wrap flex justify-left"
                      >
                        <div
                          className="xl:w-24"
                        >
                          trainee
                        </div>
                      </div>
                    </td>
                    <td
                      className="px-5 py-3 border-b border-gray-200  text-[9px] md:text-sm"
                    >
                      <p
                        className="text-red-500 whitespace-no-wrap cursor-pointer"
                        onClick={[Function]}
                      >
                        Remove
                      </p>
                    </td>
                  </tr>
                  <tr
                    className="bg-white dark:bg-dark-bg"
                  >
                    <td
                      className="border-b border-gray-200 py-2"
                    >
                      <div
                        className="flex items-left"
                      >
                        <span
                          className="hidden  ml-2 md:inline-block h-10 w-10 rounded-full overflow-hidden bg-gray-100 dark:bg-dark-tertiary"
                        >
                          <svg
                            className="h-full w-full text-gray-300 dark:text-white"
                            fill="currentColor"
                            viewBox="0 0 24 24"
                          >
                            <path
                              d="M24 20.993V24H0v-2.996A14.977 14.977 0 0112.004 15c4.904 0 9.26 2.354 11.996 5.993zM16.002 8.999a4 4 0 11-8 0 4 4 0 018 0z"
                            />
                          </svg>
                        </span>
                        <div
                          className="flex flex-col  leading-4 px-3 py-2"
                        >
                          <h3
                            className="dark:text-white sm:text-xs"
                          >
                            cyifuzo Makombe
                          </h3>
                          <p
                            className="text-sm sm:text-xs text-gray-400 dark:text-white"
                          >
                            kaka@gmail.com
                          </p>
                        </div>
                      </div>
                    </td>
                    <td
                      className="py-3 px-5 border-b border-gray-200  text-sm sm:text-xs"
                    >
                      <p
                        className="text-green-500 whitespace-no-wrap"
                      >
                        Active
                      </p>
                    </td>
                    <td
                      className="px-5 py-3 border-b border-gray-200  text-sm"
                    >
                      <div
                        className="whitespace-no-wrap flex justify-left"
                      >
                        <div
                          className="xl:w-24"
                        >
                          coordinator
                        </div>
                      </div>
                    </td>
                    <td
                      className="px-5 py-3 border-b border-gray-200  text-[9px] md:text-sm"
                    >
                      <p
                        className="text-red-500 whitespace-no-wrap cursor-pointer"
                        onClick={[Function]}
                      >
                        Remove
                      </p>
                    </td>
                  </tr>
                </tbody>
              </table>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      className="pt-10 justify-center items-center text-center"
    >
      <div
        aria-label="navigation"
      >
        <ul
          className="inline-flex -space-x-px"
        >
          <li>
            <a
              className="py-2 px-3 text-primary bg-blue-50 border border-gray-300 hover:bg-blue-100 hover:text-primary dark:border-gray-700 dark:bg-gray-700 dark:text-white"
              href="/#link"
              onClick={[Function]}
            >
              1
            </a>
          </li>
          <li>
            <a
              className="py-2 px-3 leading-tight text-gray-500 bg-white border border-gray-300 hover:bg-gray-100 hover:text-gray-700 dark:bg-gray-800 dark:border-gray-700 dark:text-gray-400 dark:hover:bg-gray-700 dark:hover:text-white"
              href="/#link"
              onClick={[Function]}
            >
              2
            </a>
          </li>
          <li>
            <a
              aria-current="page"
              className="py-2 px-3 leading-tight text-gray-500 bg-white border border-gray-300 hover:bg-gray-100 hover:text-gray-700 dark:bg-gray-800 dark:border-gray-700 dark:text-gray-400 dark:hover:bg-gray-700 dark:hover:text-white"
              href="/#link"
              onClick={[Function]}
            >
              3
            </a>
          </li>
          <li>
            <a
              className="py-2 px-3 leading-tight text-gray-500 bg-white border border-gray-300 hover:bg-gray-100 hover:text-gray-700 dark:bg-gray-800 dark:border-gray-700 dark:text-gray-400 dark:hover:bg-gray-700 dark:hover:text-white"
              href="/#link"
              onClick={[Function]}
            >
              4
            </a>
          </li>
          <li>
            <a
              className="py-2 px-3 leading-tight text-gray-500 bg-white border border-gray-300 hover:bg-gray-100 hover:text-gray-700 dark:bg-gray-800 dark:border-gray-700 dark:text-gray-400 dark:hover:bg-gray-700 dark:hover:text-white"
              href="/#link"
              onClick={[Function]}
            >
              5
            </a>
          </li>
        </ul>
      </div>
    </div>
  </div>,
]
`;
